application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -75,6 +75,22 @@
   #set ($keyboardShortcutsEnabled = false)
   #set ($discard = $xwiki.ssx.use('Diagram.DiagramSheet'))
   #set ($discard = $xwiki.jsx.use('Diagram.DiagramEditSheet'))
+  ## Issue #219: Cannot modify diagram title in diagram editor
+  &lt;div class="row xform"&gt;
+    &lt;div class="col-xs-12 col-md-12"&gt;
```

Comment:
I think the ``col-md-`` makes sense only if it has a different value than ``col-xs-``. Basically, the larger screens use by default the same styles as the smaller screens.